packages/wallet-standard/src/solana.ts

```diff
@@ -34,6 +38,8 @@ export function getNetworkForChain(chain: SolanaChain): Network {
       return Network.Mainnet;
     case SOLANA_DEVNET_CHAIN:
       return Network.Devnet;
+    case SOLANA_LOCALNET_CHAIN:
+      return Network.Localnet;
```

Comment:
Added handling for localnet back in here because it's one of the `Network` fields. The idea is that every cluster that works in Glow must be represented here or the app won't be able to detect / use them from the standard interface.

Reply:
Hmm, we don't have localnet working tho? Should we still add it? 

We don't have localnet working since the client never communicates directly with the RPC — it always hits our server.